app/scripts/metamask-controller.js

```diff
@@ -2786,6 +2787,11 @@ export default class MetamaskController extends EventEmitter {
       dismissNotifications: this.dismissNotifications.bind(this),
       markNotificationsAsRead: this.markNotificationsAsRead.bind(this),
       updateCaveat: this.updateCaveat.bind(this),
+      testOrigin: async (origin) => {
```

Comment:
Maybe this should be named something more descriptive?

Reply:
Updated. Is it better? Any other suggestions?